src/QirTools/qirlib/src/interop/pyjit/gates.rs

```diff
@@ -0,0 +1,216 @@
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(non_snake_case)]
+#![allow(dead_code)]
```

Comment:
The other `allow` attributes are fairly straightforward, but it may help to add a comment explaining why allowing dead code is helpful.